internal/common/echo_logrus.go

```diff
@@ -0,0 +1,190 @@
+package common
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/labstack/gommon/log"
+	"github.com/sirupsen/logrus"
+)
+
+// EchoLogrusLogger extend logrus.Logger
+type EchoLogrusLogger struct {
+	*logrus.Logger
+}
+
+var commonLogger = &EchoLogrusLogger{
+	Logger: logrus.StandardLogger(),
+}
+
+func Logger() *EchoLogrusLogger {
+	return commonLogger
+}
+
+func toLogrusLevel(level log.Lvl) logrus.Level {
+	switch level {
+	case log.DEBUG:
+		return logrus.DebugLevel
+	case log.INFO:
+		return logrus.InfoLevel
+	case log.WARN:
+		return logrus.WarnLevel
+	case log.ERROR:
+		return logrus.ErrorLevel
+	}
+
+	return logrus.InfoLevel
+}
+
+func toEchoLevel(level logrus.Level) log.Lvl {
+	switch level {
+	case logrus.DebugLevel:
+		return log.DEBUG
+	case logrus.InfoLevel:
+		return log.INFO
+	case logrus.WarnLevel:
+		return log.WARN
+	case logrus.ErrorLevel:
+		return log.ERROR
+	}
+
+	return log.OFF
+}
+
+func (l *EchoLogrusLogger) Output() io.Writer {
+	return l.Out
+}
+
+func (l *EchoLogrusLogger) SetOutput(w io.Writer) {
+	l.Out = w
+}
+
+func (l *EchoLogrusLogger) Level() log.Lvl {
+	return toEchoLevel(l.Logger.Level)
+}
+
+func (l *EchoLogrusLogger) SetLevel(v log.Lvl) {
+	l.Logger.Level = toLogrusLevel(v)
```

Comment:
Same I don't think we should set the global logger via echo. All the configuring should happen in cmd/. So imo this should be a nop.